spec/support/simple_report.rb

```diff
@@ -61,3 +61,12 @@ def param_name
   end
 
 end
+
+module Ns45
+  class TranslatedReport
```

Comment:
I preffer to define classes that are only used in 2-3 tests in place. It makes it more obvious what it is designed for and let it server only one test purpose instead of reusing classes between tests (it was like that before - it was terrible idea).